bde_fdw_rds/bde_fdw_rds_stack.py

```diff
@@ -0,0 +1,28 @@
+from aws_cdk import Stack, aws_ec2
+from constructs import Construct
+
+
+class BdeFdwRdsStack(Stack):
+    def __init__(  # type: ignore[no-untyped-def]
+        self,
+        scope: Construct,
+        construct_id: str,
+        vpc_id: str,  # vpc id
+        subnet_ids: list[str],  # list of subnet ids
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # ----- Networking -----
+        aws_ec2.Vpc.from_lookup(self, "ExistingVPC", vpc_id=vpc_id)
+        subnets = []
+        for subnet_id in subnet_ids:
+            subnets.append(
+                aws_ec2.Subnet.from_subnet_attributes(
+                    self,
+                    subnet_id.replace("-", "").replace("_", "").replace(" ", ""),
```

Comment:
Why is this cleanup needed?

Reply:
Good question. I followed the standard pattern here (https://github.com/aws-samples/aws-cdk-examples/blob/master/python/rds/mysql/mysql.py#L26), but should probably look into the inner workings on why it is needed.